ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -82,6 +82,19 @@
             $ 100.00
           </div>
         </li>
+        <li
+          v-if="hasCertifyFee"
+          id="certify-fee"
+          :class="[$style['fee-container'], $style['fee-list__item'], 'pb-4', 'pr-4', 'py-4']"
```

Comment:
Can we use regular **scoped** css classes? e.g. `:class="fee-container"`

Reply:
I don't think this could be changed without redoing all of the css for this component...

Reply:
This is how the other rows in the fee summary are formatted in this component

Reply:
Sure, if it's a large effort, then I guess we can do tech debt cleanup later, if ever. I was hoping to do it gradually.